src/models/ytelse/foreldrepenger.ts

```diff
@@ -24,14 +24,8 @@ export interface Foreldrepengerettighet {
     arbeidsforhold: Arbeidsforhold[];
     erArbeidsgiverperiode: boolean | null;
     arbeidskategori: string | null;
-}
-
-export interface Fødsel extends Foreldrepengerettighet {
-    termin: String;
-}
-
-export interface Adopsjon extends Foreldrepengerettighet {
-    omsorgsovertakelse: String;
+    termin: string | null;
```

Comment:
Betyr ikke dette at objektet teknisk sett kan ha både termin OG omsorgsovertakelse? Make the impossible impossible! 💃

Reply:
👌